mtags/src/main/scala-2/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -323,7 +336,12 @@ class CompletionProvider(
     else if (symbol.isClass) k.Class
     else if (symbol.isMethod) k.Method
     else if (symbol.isCaseAccessor) k.Field
-    else if (symbol.isVal && !symbolIsFunction) k.Value
+    else if (symbol.isVal && !symbolIsFunction)
+      r match {
```

Comment:
I know this was here before, but since you're using it, may I propose instead of using `r` we just use `member` as the argument name for `completionItemKind`?

Reply:
Fixed!